From 230a21e9869034d34e6d07cb042fa37e7e638cb7 Mon Sep 17 00:00:00 2001
From: John Jacques <john.jacques@intel.com>
Date: Mon, 4 May 2026 10:29:16 -0500
Subject: lib/crc: Add CRC8 Config Option

CRC8 is only available as a module when another
built-in module selects it.  To allow external modules
to use it, make it available always.

Signed-off-by: John Jacques <john.jacques@intel.com>
---
 lib/crc/Kconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/lib/crc/Kconfig b/lib/crc/Kconfig
index 70e7a6016de3..8ba767cf0aa5 100644
--- a/lib/crc/Kconfig
+++ b/lib/crc/Kconfig
@@ -15,7 +15,8 @@ config CRC7
 	  the functions from <linux/crc7.h>.
 
 config CRC8
-	tristate
+	tristate "Include the CRC8 Library Functions"
+	default n
 	help
 	  The CRC8 library functions.  Select this if your module uses any of
 	  the functions from <linux/crc8.h>.
-- 
2.25.1

